extensions/ql-vscode/test/vscode-tests/no-workspace/codeql-cli/distribution.test.ts

```diff
@@ -286,7 +286,7 @@ describe("Launcher path", () => {
     const manager = new DistributionManager(
       { customCodeQlPath: pathToCmd } as any,
       {} as any,
-      undefined as any,
+      {} as any,
```

Comment:
I'm not hugely happy that we're doing `as any` in the tests, but I don't think the change in this PR makes it any worse than it was already.